source/tls_channel_handler.c

```diff
@@ -62,82 +62,49 @@ void aws_tls_ctx_options_clean_up(struct aws_tls_ctx_options *options) {
     AWS_ZERO_STRUCT(*options);
 }
 
-static int s_load_null_terminated_buffer_from_cursor(
```

Comment:
We don't need this function anymore.
A while back Bret made is so we turn this into a null-terminated string right before passing it into s2n, instead of fastidiously trying to keep an untracked 0 at the end of the aws_byte_buf in N different places